server/controllers/auth/forgot-password.js

```diff
@@ -46,8 +43,18 @@ async function forgotPasswordController(req, res, next) {
         statusCode: 503,
       });
 
-    const mail = mailText(userToken.tokenURL());
-    const nodeMailerRes = await sendEmail(user.email, mail.subject, mail.body);
+    const htmlFile = fs.readFileSync(
+      __dirname + "/../../templates/ForgotPasswordOrVerify.html",
+      "utf-8"
+    ).toString();
+    const template = handlebars.compile(htmlFile);
+    const replacements = {
+      url: userToken.tokenURL(),
+      message1: "It seems like you forgot your password?",
```

Comment:
Let make text dynamic or add in contants.

Reply:
This is only used in this file. Moving it to constants will make sense only when this is used at several places